app/services/feature_flag.rb

```diff
@@ -39,6 +40,10 @@ def feature
     [:account_and_org_settings_changes, 'Allows new account and org setting changes', 'Despo Pentara'],
   ].freeze
 
+  FEATURE_TYPES = {
+    send_request_data_to_bigquery: 'permanent',
```

Comment:
Could we do something that makes the flags in the PERMANENT_SETTINGS list be marked as permanent automatically?

Reply:
The thing is I don't think we want to mark all those as permanent. For instance the `dfe_sign_in_fallback`, can be on/off for different envs if an issue happens, if we forgot to turn it off after the incident is complete I'd like to be alerted about that. I think the only candidate I could see for now is big query, but we can expand the list appropriately

Reply:
It might be a bit confusing since the term 'permanent' is overloaded now. Is there a need for a type, or could we just mark certain flags as "ignored by the dashboard"?